candig/server/backend.py

```diff
@@ -586,7 +586,7 @@ def countHelper(self, table, fv_counts):
         """
         response_list = []
         if fv_counts:
-            if self._dpEpsilon:
+            if self._dpEpsilon and float(self._dpEpsilon) != 0.0:
```

Comment:
I am not familiar with this part, under which circumstances will the `self._dpEpsilon` be `0.0`? Is it when it's being set to 0 explicitly?

Reply:
Yea, but I am mostly trying to avoid a runtime error here. I can remove it if you want.

Reply:
No, it's good to have more checks.. the runtime error you mentioned is related to when `self._dpEpsilon` is set explicitly to `0`, right? @amanjeev

Reply:
I am talking about this runtime error, setting it to zero, even though that is unlikely.

Reply:
Makes sense. Thanks for the clarification!